.github/scripts/setup_runners_service_rpm.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+echo "install required packages"
+sudo yum install python37 git unzip wget -y
+
+REPO_ROOT=`git rev-parse --show-toplevel`
+ROOT=`dirname $(realpath $0)`; cd $ROOT
+OD_VERSION=`python $REPO_ROOT/bin/version-info --od`
+SETUP_ACTION=$1
+
+echo "############################################################"
+echo "Setup ES/KIBANA to start with YES/NO security configurations"
+echo "############################################################"
+
+echo "setup parameters"
+echo $JAVA_HOME
+sudo curl https://d3g5vo6xdbdb9a.cloudfront.net/yum/staging-opendistroforelasticsearch-artifacts.repo  -o /etc/yum.repos.d/staging-opendistroforelasticsearch-artifacts.repo
+export PATH=$PATH:$JAVA_HOME
+
+#####################################################################################################
+
+echo "install es"
+sudo yum install opendistroforelasticsearch-$OD_VERSION -y
+sudo mkdir -p /home/repo
+sudo chmod 777 /home/repo
+sudo chmod 777 /etc/elasticsearch/elasticsearch.yml
+sudo sed -i '/path.logs/a path.repo: ["/home/repo"]' /etc/elasticsearch/elasticsearch.yml
+sudo sed -i /^node.max_local_storage_nodes/d /etc/elasticsearch/elasticsearch.yml
```

Comment:
Do we also need to add "sudo sysctl -w vm.max_map_count=262144".
Had seems some issues with ES before

Reply:
Resolved and completely refactoring the setup_runner_service.sh to work on tar/deb/rpm distros.
Thanks.

Reply:
Did you add that command? Unable to see it

Reply:
@gaiksaya The file has a name change and also refactorting.
https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/b3d6ead505804506cdc392b379e4b1ccf3387b3b/.github/scripts/setup_runners_service.sh#L33